doc/admin-guide/introduction.en.rst

```diff
@@ -187,14 +187,6 @@ requests and manage, control, and monitor the health of the system.
    first-served order. This connection queueing shields users from any
    server restart downtime.
 
-The figure below illustrates the |TS| processes.
-
-.. figure:: ../static/images/admin/process.jpg
```

Comment:
We should also remove the image since is it not relevant.

Reply:
Good catch. Thanks for the review.

Reply:
I am not seeing the image file removed in the last git push.